src/d3a/models/strategy/area_agents/two_sided_pay_as_clear_engine.py

```diff
@@ -167,5 +172,5 @@ def tick(self, *, area):
         self.mcp_update_point = \
             area.config.ticks_per_slot / \
             ConstSettings.GeneralSettings.MARKET_CLEARING_FREQUENCY_PER_SLOT
-        if current_tick_number % int(self.mcp_update_point) == 0:
+        if (current_tick_number+1) % int(self.mcp_update_point) == 0 and current_tick_number != 0:
```

Comment:
I understand this part: `and current_tick_number != 0`
But why the +1 for current_tick_number ?

Reply:
current_tick_number = 0 is the first tick, last tick is 59 or 60th